doc/extending/extending_theano.txt

```diff
@@ -1,24 +1,35 @@
 
-.. _extending_theano:
```

Comment:
Should we keep that label so that the file can more easily be referenced from the table of content?